[2024-05-30T12:13:42.405+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T12:13:42.430+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:13:17.266392+00:00 [queued]>
[2024-05-30T12:13:42.441+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:13:17.266392+00:00 [queued]>
[2024-05-30T12:13:42.443+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-30T12:13:42.457+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): bash_task_end> on 2024-05-30 12:13:17.266392+00:00
[2024-05-30T12:13:42.468+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1111) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-30T12:13:42.470+0000] {standard_task_runner.py:63} INFO - Started process 1113 to run task
[2024-05-30T12:13:42.469+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'bash_task_end', 'manual__2024-05-30T12:13:17.266392+00:00', '--job-id', '355', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpa0pj43z7']
[2024-05-30T12:13:42.473+0000] {standard_task_runner.py:91} INFO - Job 355: Subtask bash_task_end
[2024-05-30T12:13:42.530+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:13:17.266392+00:00 [running]> on host 29c09f03cc50
[2024-05-30T12:13:42.618+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='bash_task_end' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T12:13:17.266392+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T12:13:17.266392+00:00'
[2024-05-30T12:13:42.619+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T12:13:42.633+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-30T12:13:42.635+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here is your bash message at the end of your DAG"']
[2024-05-30T12:13:42.645+0000] {subprocess.py:86} INFO - Output:
[2024-05-30T12:13:42.646+0000] {subprocess.py:93} INFO - Here is your bash message at the end of your DAG
[2024-05-30T12:13:42.647+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-30T12:13:42.648+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T12:13:42.674+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=bash_task_end, execution_date=20240530T121317, start_date=20240530T121342, end_date=20240530T121342
[2024-05-30T12:13:42.725+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T12:13:42.743+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T12:13:42.747+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
